src/SFA.DAS.AssessorService.Web/Validators/CertificateNameViewModelValidator.cs

```diff
@@ -0,0 +1,21 @@
+﻿using FluentValidation;
+using SFA.DAS.AssessorService.Web.ViewModels.Certificate;
+
+namespace SFA.DAS.AssessorService.Web.Validators
```

Comment:
There is a problem with this, the previous name is being compared to the new name to see if only the case has differed - but as the previous name is being stored in a hidden field it is possible to construct a POST such that you change both the previous and the old names and allow yourself to change the name to anything - the previous name should not be stored in the view instead the certificate id should be stored and then in the validation rule lookup the certificate current name from the database and do the comparison to see if only the case has changed between that and the new name